modules/scripts/startup-script/variables.tf

```diff
@@ -263,8 +263,8 @@ variable "http_no_proxy" {
   nullable    = false
 }
 
-variable "enable_cloud_rdma" {
-  description = "If true, will install and enable Cloud RDMA drivers. Currently only supported on Rocky Linux 8."
+variable "install_cloud_rdma" {
```

Comment:
Maybe make this `install_cloud_rdma_drivers` so it's clear exactly what this flag does.

Reply:
Added, also changed the script name to match so intention is clear for this variable/script.